connect-widget/app/src/pages/connect/components/error-dialog.tsx

```diff
@@ -32,7 +33,14 @@ const ErrorDialog = ({ show, title, message, link, onClose: closed }: ErrorDialo
   });
 
   useEffect(() => {
-    isOpen && capture.message("Error dialog displayed", { extra: { show, title, message, link } });
+    if (
+      title !== NoTokenError.DEFAULT_TITLE &&
+      title !== DemoTokenError.DEFAULT_TITLE &&
+      title !== InvalidTokenError.DEFAULT_TITLE
+    ) {
+      isOpen &&
+        capture.message("Error dialog displayed", { extra: { show, title, message, link } });
+    }
```

Comment:
I should've thought about it before, but just so you know how Sentry works:
- based on the message we send to it (either `capture.error(new Error("<msg>"), ...` or `capture.message("<msg>"...`) it will create a Sentry issue
- we can ignore/assign/etc on each issue
- the problem is that all things coming through the ErrorDialog are getting the same message "Error dialog displayed" and thus, associated to the same Sentry issue
   - this means we can't ignore notifications for these, otherwise we can't know when a use actually has an unknown problem
- ideally, we adjust notifications/ignore, instead of filtering what we send to Sentry (this way we have more information about what's going on with our app through Sentry)

With that being said, maybe we can still report to Sentry but with a message specific for each situation, maybe the title? I'd just keep the "Error dialog displayed" message if not one of those Errors you added to the conditional/if.

Reply:
Oh I see! Now this makes sense!